backend/db/index.js

```diff
@@ -26,8 +26,7 @@ const findOneFromDB = (findCondition, collectionName) => {
 };
 
 const updateOneFromDb = async (condition, data, collectionName) => {
```

Comment:
Even async/await is not required. It can be simply: 
```
const updateOneFromDb = (condition, data, collectionName) => db.collection(collectionName).updateOne(condition, { $set: data })
```

Reply:
Done on [commit id](71cf724c5618f3f140a2676a4e57d28c98a499f1)